README.md

```diff
@@ -1,2 +1,17 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Create a list of random words which the same length
+Choose a word from the list to be the key word
+Guessed word = false
+Attempts =0
+While Guessed word = false
+For attempts in range 4
+		Guess a word from the list
+		Attempts = attempts +1
+		If guessed word = key word
+		Complete hack
+		End
+		If guessed word != key word and attempts = 4
+		Failed hack
+		end
+![flowchart](https://github.com/proarua15151/comp110-worksheet-B/blob/master/fallout%204%20hacking%20flowchart.pdf)
```

Comment:
Flowchart looks mostly OK. However the user choosing a word is not included in the loop, so as it stands the game will simply check the same guess repeatedly rather than asking for another. Tracing the flowchart by hand would help to catch issues of this type (and indeed is also useful when debugging code)